_locales/en/messages.json

```diff
@@ -969,6 +969,11 @@
   "supportDescription": "Having issues? Explore help articles or open a ticket with Session Support.",
   "supportGoTo": "Go to Support Page",
   "systemInformationDesktop": "System Information: {information}",
+  "settingsCannotChangeDesktop": "Cannot Update Setting",
+  "settingsStartCategoryDesktop": "Startup",
+  "launchOnStartDesktop": "Launch on Startup",
+  "launchOnStartDescriptionDesktop": "Launch Session automatically when your computer starts up.",
+  "launchOnStartupDisabledDesktop": "This setting is managed by your system on Linux. To have Session start automatically, please add it to your startup applications in your system settings.",
```

Comment:
avoid commiting those, we want those to come from crowdin only. 
The flow should be 
- commit all but those changes, even if that means the CI will fail
- wait for the strings to be added to crowdin
- merge the crowdin PR to your branch (you get the new strings)
- the CI should now  be fixed